### PR TITLE
Added to the description of registering, updating and releasing apps

### PR DESCRIPTION
**Pull request by landml**
@jayrbolton, @JamesJeffryes  - I updated the version in conf.py to be the same version as the SDK. I got the proper format for the publications from Mikayla because she just updated all the apps. This was added to the document your app. I added a bit on setting the MAINTAINER in the Dockerfile. Steve pointed it out to me when I tried update as app. I added a link to what is the KBase narrative interface for those that are new. I was a comment that came back for the SFAs. Pulled out all the ... lines from the implement.rst file because make check didn't like them.

### CODE REVIEW

File: source/howtos/edit_your_dockerfile.rst [hunk 1]
```diff
@@ -7,6 +7,16 @@ up your container.
 
 A helpful resource is the |bestPractices_link| guide.
 
+The first two lines of the Dockerfile look something like:
+
+:: 
+
+    FROM kbase/kbase:sdkbase2.latest
```
Comment (jayrbolton) [inline source/howtos/edit_your_dockerfile.rst]: Should this actually be `kbase/sdkbase2:python`?

**Comment by jayrbolton** [comment]
@landml I only had a couple minor requests/questions above

File: source/howtos/edit_your_dockerfile.rst (hunk 1 above)
Comment (landml) [reply source/howtos/edit_your_dockerfile.rst]: After I wrote the above, I read in slack that .latest was for python 2 and .python was for python3

**Comment by landml** [comment]
@jayrbolton - I only saw one comment. Was there anything else you would like me to update?

**Comment by jayrbolton** [comment]
Just that the "app_catalog.png" image doesn't seem to be used anywhere, not sure if you meant that. Otherwise it looks good to me on my laptop

**Comment by landml** [comment]
I don't know where "app_catalog.png" was and what I moved around but I'll try to work in for the next version. 😉

**Pull request merged**